pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -587,10 +587,10 @@ var _ = Describe("Defaults", func() {
 		})
 
 		It("should default cri.name to containerd when worker Kubernetes version >= 1.22", func() {
-			obj.Spec.Kubernetes.Version = "1.18"
+			obj.Spec.Kubernetes.Version = "1.22"
```

Comment:
```suggestion
			obj.Spec.Kubernetes.Version = "1.21"
```

We could leave the K8s version < 1.22 to make sure that the worker K8s versions gets respected and that's why the worker CRI is updated to containerd. And let's keep the worker K8s version to be 1.22 as before.